tests/src/test/scala/com/nvidia/spark/rapids/HashAggregatesSuite.scala

```diff
@@ -46,7 +46,8 @@ class HashAggregatesSuite extends SparkQueryCompareTestSuite {
           case _ => None
         }
       }
-      assert(gpuAgg.isDefined, "as the GPU plan expected a GPU aggregate but did not find any!")
+      assert(gpuAgg.isDefined,
+        s"as the GPU plan expected a GPU aggregate but did not find any! Plan: $plan")
```

Comment:
Shouldn't this be looking at `executedPlan` rather than `plan`?  Or would it be helpful to show both if something weird is going on?

Reply:
Given the uncertainty on this one, the more information we have the better, so I have added both plans to the message now.